react/src/App.tsx

```diff
@@ -60,15 +47,36 @@ function App() {
               sapien, ac bibendum nisl aliquam in. Morbi ac velit et ligula
               consectetur interdum. Vestibulum condimentum, augue vitae lobortis
               rhoncus, mi est ultricies mi, sed tincidunt magna nibh in lectus.
-              Pellentesque vel vulputate orci, vel lacinia orci. Sed suscipit
-              leo at diam ullamcorper, vitae volutpat neque dapibus. Maecenas
-              sit amet rhoncus arcu. Sed sed molestie elit. Nullam in interdum
-              est, vitae aliquam ipsum. Nunc rutrum nibh ut arcu egestas
-              egestas.
+              Pellentesque vel vulputate orci, vel lacinia orci. Sed suscipit leo
+              at diam ullamcorper, vitae volutpat neque dapibus. Maecenas sit amet
+              rhoncus arcu. Sed sed molestie elit. Nullam in interdum est, vitae
+              aliquam ipsum. Nunc rutrum nibh ut arcu egestas egestas.
             </p>
+            <sdds-block>
+              <div className="sdds-headline-01">Headline</div>
+              <p className="sdds-body-01">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+                condimentum nisi ut eleifend ultrices. Nunc venenatis maximus
+                sapien, ac bibendum nisl aliquam in. Morbi ac velit et ligula
+                consectetur interdum. Vestibulum condimentum, augue vitae lobortis
+                rhoncus, mi est ultricies mi, sed tincidunt magna nibh in lectus.
+                Pellentesque vel vulputate orci, vel lacinia orci. Sed suscipit
+                leo at diam ullamcorper, vitae volutpat neque dapibus. Maecenas
+                sit amet rhoncus arcu. Sed sed molestie elit. Nullam in interdum
+                est, vitae aliquam ipsum. Nunc rutrum nibh ut arcu egestas
+                egestas.
+              </p>
+            </sdds-block>
           </sdds-block>
-        </sdds-block>
         </div>
+        <div>
+        <div className="sdds-headline-02 sdds-u-pb1">Breadcrumbs</div>
+        <sdds-breadcrumb>
+          <sdds-breadcrumb-item href="#">Step one</sdds-breadcrumb-item>
+          <sdds-breadcrumb-item href="www.google.se">Step two</sdds-breadcrumb-item>
+          <sdds-breadcrumb-item current={true}>Current page</sdds-breadcrumb-item>
```

Comment:
WebStorm was suggesting to assign a value for the "current" attribute so I just added true. Will this be the case for all boolean props in React, do you know more about it @mJarsater ?

Reply:
It won't be needed for boolean props as long as they are true. Just setting <sdds-breadcrumb-item current tells the component that current is true, and not setting it will say that is not.